Builder/SolariumSelectQueryBuilder.php

```diff
@@ -71,33 +64,35 @@ public function buildSolariumQueryFromGeneric(GenericSelectQuery $query)
         }
 
         //determine whether we are using the facet values for an underlying "base" (recorded) query
-        $shouldUseFacetValuesForRecordedQuery = (bool) $this->hasSearchContext()
-            && $query instanceof RecordableSelectQueryInterface
+        $shouldUseFacetValuesForRecordedQuery = (bool) $query instanceof RecordableSelectQueryInterface
```

Comment:
I couldn't determine why it was important/relevant that there was an applied search context while working out this value. I thought it was safe to remove it (since the context is no longer available in this class - by design).